README.md

```diff
@@ -1,2 +1,66 @@
-# BVGOG-Contract-Manager
-Full-Stack source code for BVCOG Contract Management Platform
\ No newline at end of file
+<!-- Github Actions Build Status -->
+![Build Status](https://github.com/BVCOG-Contract-Management/BVGOG-Contract-Manager/actions/workflows/ruby.yml/badge.svg)
+
+<img src="./app/assets/images/bvcog-logo.png" alt="BVCOG Logo" width="200"/>
+
+# BVCOG Contract Management System
+Full-Stack source code for the BVCOG Contract Management Platform (CMS)
+
+## Getting Started
+### Prerequisites
+* [Ruby](https://www.ruby-lang.org/en/downloads/) - Ruby 3.2.0
+* [Rails](https://rubyonrails.org/) - Rails
+* [PostgreSQL](https://www.postgresql.org/) - PostgreSQL 12.3
+
+### Installing for Local Development
+1. Clone the repository
+```
+git clone
+```
+2. Install dependencies
+```
+bundle install
```

Comment:
One suggestion: `bundle config set --local without 'production'` and then `bundle install`